www/src/components/layout/top-banner.js

```diff
@@ -0,0 +1,71 @@
+import React from "react"
+import PropTypes from "prop-types"
+import styled from "react-emotion"
+
+import presets, { colors } from "../../utils/presets"
+import { rhythm, scale, options } from "../../utils/typography"
+
+const horizontalPadding = rhythm(1 / 2)
+const backgroundColor = props =>
+  props.background ? props.background : colors.gatsby
+
+const TopBannerContainer = styled("div")(props => ({
+  backgroundColor: backgroundColor(props),
```

Comment:
To keep these using plain CSS whenever possible, another way to write this would be:

```js
const TopBannerContainer = styled('div')`
	background-color: ${props => backgroundColor(props)};
	height: ${presets.bannerHeight};
	/* ... */
`;
```

This is definitely a nitpick, but my intention is to improve consistency — CSS is challenging as-is, so I'd like to avoid asking maintainers to remember when to use `margin-right` vs. `marginRight`, etc.

Reply:
+1

Reply:
@jlengstorf , so now, I have to commit fixes to the PR, am I right?

Reply:
@greglobinski Yes, just add commits to the PR branch!